editor/src/components/canvas/__snapshots__/ui-jsx-canvas.spec.tsx.snap

```diff
@@ -47297,10 +47297,7 @@ exports[`UiJsxCanvas render renders correctly when a component is passed in via
           style=\\"background-color: pink; width: 100%; height: 20px\\"
           data-path=\\"eee/fff/app-entity:aaa/bbb:ddd\\"
         >
-          <div
-            data-uid=\\"ccc~~~1\\"
-            data-path=\\"eee/fff/app-entity:aaa/bbb/ccc~~~1\\"
-          >
+          <div data-uid=\\"ccc\\" data-path=\\"eee/fff/app-entity:aaa/bbb/ccc\\">
```

Comment:
is this change desired here?

Reply:
Yes. Before this jsx render props were wrapped into an expression (which just returned the element), which made them a generated element.
After this change these elements will not be considered generated (which is a prerequisite to make them editable).